coss/opensource_clubs/models.py

```diff
@@ -117,9 +148,19 @@ class EntityPageTag(TaggedItemBase):
 
 class EntityDetailPage(Page):
     """Actual page for the Open Source clubs."""
+
+    entity_title = models.CharField(max_length=128, default='')
     description = fields.RichTextField(blank=True)
     featured = models.BooleanField(default=False)
+    is_flagged = models.BooleanField(default=False)
+    club_name = models.CharField(max_length=128, default='')
```

Comment:
Is this different than the `title`?

Reply:
this is the `campus/club name` that exists in the design.